packages/scaaml_dataset/tests/io/test_example_iterator.py

```diff
@@ -18,7 +18,7 @@
 import numpy as np
 
 from scaaml.io import Dataset
-from scaaml.stats import ExampleIterator
+from scaaml_dataset.io import ExampleIterator
```

Comment:
I'm not sure about best practice here.
But maybe we can consider scaaml-dataset as being a package belonging to the overall scaaml namespace.
Each package would contribute with its own files.

But maybe that's too complicated to setup when things aren't split across multiple repositories.

Reply:
I didn't know about namespaces. So not sure about best practices at all.

Just to be sure in the end we want to have separate repositories, but instead of:

from scaaml_dataset.io import ExampleIterator

we would write:

from scaaml.dataset.io import ExampleIterator

The second one looks nicer, but I like that the first one directly gives you the package name. Also scaaml-dataset is not taken as a package name, no idea how naming works on pypi with namespaces.